test/integration/services.js

```diff
@@ -23,12 +23,12 @@ describe('app', () => {
           expect($('.services').text()).to.equal('Services at this surgery');
           expect($('.services').next('h3').text()).to.equal('Services patients can book themselves');
           expect($('.self-referrals li').length).to.equal(6);
-          expect($($('.self-referrals li')[0]).text()).to.equal('Asthma clinic');
```

Comment:
`The integration tests are asserting that data held in the image is being rendered, and is less concerned with what the data is.` - I disagree with this, I think the service mapping is a `feature` of the whole application and it not being reflected in the ES image is not right.